cmd/kubeadm/app/phases/upgrade/policy_test.go

```diff
@@ -231,3 +231,74 @@ func TestEnforceVersionPolicies(t *testing.T) {
 		})
 	}
 }
+
+func Test_detectUnstableVersionError(t *testing.T) {
```

Comment:
Thanks @my-git9 
But it seems that the above `TestEnforceVersionPolicies` has covered the test cases about detectUnstableVersionError.

Reply:
Thanks for reminding, I will close it